presubmit-service/check_if_pr_can_cherry_pick.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+
+# The organization containing the repo to be checked against, e.g. angular
+ORG=$1
+# The repo to be checked against, e.g. material2
+REPO=$2
+# The PR number to retrieve the patch files for, e.g. 1234
+PR=$3;
+# The branch in the repo to check for cherry-picking onto, e.g. 6.1.x
+BRANCH=$4;
+# The fullname of the repository to be checked against, e.g. angular/material2
+FULL_REPO="$ORG/$REPO"
+# Whether the PR can be cherry-picked on to the targetted branch.
+RESULT=false;
+
+# If the needed tmp directories don't exist, create them.
+[ ! -d "/tmp/presubmit_service/$FULL_REPO/" ] && mkdir -p /tmp/presubmit_service/$FULL_REPO;
+
+# If the local_repo directory does not exist, create it.
+[ ! -d ".presubmit_service/local_repo" ] && mkdir -p .presubmit_service/local_repo;
+cd .presubmit_service/local_repo;
+
+# If the current directory, which should be repos is not a toplevel of a git repo
+# initialize a repo and create a dummy commit in to create a local remote with a
+# master branch.
+if [ "$(git rev-parse --show-toplevel)" != "$(pwd)" ]; then
+  git init 2>&1;
```

Comment:
Would it make sense to redirect stderr to stdout for the entire script instead of for each command within the script?

Reply:
Paul and I were talking about this above, long term the plan is to have specific `stdout`/`stderr` to provide output to the `shelljs.exec` method so that it can give more context in the response.

For now, we just are having everything logged this way so it shows up in AppEngine's logs.

Reply:
Add a TODO?